cypress/e2e/course/addBBBToolToCourseInNBC.feature

```diff
@@ -0,0 +1,54 @@
+@release
+Feature: Teacher can add and remove BBB tool in the course in NBC with admin permission
+
+  As a teacher I want to activate and deactivate BBB tool inside the course in NBC
+
+  @stable_test
+  Scenario Outline: Admin enables the video conference in the school settings page
+    Given I am logged in as a 'admin1_nbc' at 'nbc'
+    When I go to administration page
+    When I click on manage school card
+    #remove the following line if old admin page is hidden
+    When I click on general settings panel
+    Then I enable the video conference
+    Then I click on admin setting save button
```

Comment:
should be "I click on button Save admin settings"

But I already didn't change it, I think it's used multiple ...

Reply:
Nevertheless, I updated the feature line.